shared/js/injected-content-scripts/fingerprint-protection.js

```diff
@@ -4,7 +4,7 @@
  * data through obsufcation / randomness.
  */
 
-(function protect () {
+(async function protect () {
```

Comment:
Does making this async make any of the protection timing issues worse here?

Reply:
By definition, yeah there will at least be one tick of JS that is worse; The browser APIs aren't really clear on how long a local fetch will take but it's certainly better not to do that. However I think using the build script as mentioned to combine all the content-scope scripts into this file fixes it anyway.